HomeTask1/MatrixMultiplication/Matrix.cs

```diff
@@ -0,0 +1,238 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+using System.Threading;
+
+namespace MatrixMultiplication
+{
+    /// <summary>
+    /// Матрицы типа int размера n * m
+    /// </summary>
+    public class Matrix
+    {
+        private int[,] mas;
+        private int _strings;
+        /// <summary>
+        /// Кколичество строк в матрице
+        /// </summary>
+        public int Strings
+        {
+            get 
+            {
+                return _strings; 
+            }
+            set 
+            {
+                _strings = value; 
+            }
+        }
```

Comment:
```suggestion
        public int Strings {get; set; }
```
И тогда поле _strings будет не нужно.